pavement.py

```diff
@@ -73,9 +73,22 @@ def make_venv():
     bootstrap_venv(__here__ / path('env'), 'sideboard')
     develop_sideboard()
 
+def install_pip_requirements_in_dir(dir_of_requirements_txt):
+    path_to_pip = __here__ / path('env/bin/pip')
+    sh('{pip} install -e {dir_of_requirements_txt}'
```

Comment:
@EliAndrewC I changed this path from 'pip3' to 'pip', so should be solid for python2